doc-src/ch-unmountsqfs_desc.rst

```diff
@@ -0,0 +1,53 @@
+Synopsis
+========
+
+::
+
+  $ ch-unmountsqfs MOUNTDIR
+
+Description
+===========
+
+Unmounts Charliecloud squashfs at target directory :code:`MOUNTDIR`
+Removes empty :code:`MOUNTDIR` after successful unmounting
```

Comment:
Should we remove `MOUNTDIR` afterwards? Definitely the wrong thing to do if we didn't create it in `ch-mountsqfs` but definitely the right thing to do if we did? I wonder if the UI should mirror `ch-tar2dir` more closely (argument is parent directory, always create, always remove)?

Reply:
I may open an issue to gather input on this subject to make the behavior of both scripts consistent.

Reply:
I am biased to option one, because it is easier (in my own mind) to handle in scripts.

`./myscript.sh /home/me/hello.sqfs /var/tmp/hello`

That way users would only need to pass in the full path to the mount, instead of having to predict the output of the mount in their scripts.